### ISSUE TITLE
2021-10

### ISSUE THREAD

**Issue by qiaojy19** [qiaojy19/q-Daily#9]
# Oct - 2021

## 10.25 - 10.29
| Title |
| ----- |
Accumulative Poisoning Attacks on Real-time Data
Adversarial Robustness through Bias Variance Decomposition: A New Perspective for Federated Learning
ARIANN: Low-Interaction Privacy-Preserving Deep Learning via Function Secret Sharing
Blockchain-based Trustworthy Federated Learning Architecture
Communication-Efficient ADMM-based Federated Learning
Computational Intelligence and Deep Learning for Next-Generation Edge-Enabled Industrial IoT
DAdaQuant: Doubly-adaptive quantization for communication-efficient Federated Learning
DeceFL: A Principled Decentralized Federated Learning Framework
DFL: High-Performance Blockchain-Based Federated Learning
Differentially Private Federated Bayesian Optimization with Distributed Exploration
DPCOVID: Privacy-Preserving Federated Covid-19 Detection
Dynamic Differential-Privacy Preserving SGD
Early Detection of COVID-19 Hotspots Using Spatio-Temporal Data
Efficient passive membership inference attack in federated learning
Ensemble Federated Adversarial Training with Non-IID data
Faithful Edge Federated Learning: Scalability and Privacy
Fault-Tolerant Federated Reinforcement Learning with Theoretical Guarantee
FedDR -- Randomized Douglas-Rachford Splitting Algorithms for Nonconvex Federated Composite Optimization
Federated Learning for Malware Detection in IoT Devices
Federated Learning on Non-IID Data Silos: An Experimental Study
Federated Multi-Task Learning under a Mixture of Distributions
Federated Reconstruction: Partially Local Federated Learning
Federated Semi-Supervised Learning with Class Distribution Mismatch
FedPara: Low-Rank Hadamard Product for Communication-Efficient Federated Learning
FedPrune: Towards Inclusive Federated Learning
FeO2: Federated Learning with Opt-Out Differential Privacy
Fine-tuning in Federated Learning: a simple but tough-to-beat baseline
FLTrust: Byzantine-robust Federated Learning via Trust Bootstrapping
FL-WBC: Enhancing Robustness against Model Poisoning Attacks in Federated Learning from a Client Perspective
Gradient Inversion with Generative Image Prior
HyFed: A Hybrid Federated Framework for Privacy-preserving Machine Learning
Implicit Gradient Alignment in Distributed and Federated Learning
Improving Fairness via Federated Learning
Local and Central Differential Privacy for Robustness and Privacy in Federated Learning
Local Model Poisoning Attacks to Byzantine-Robust Federated Learning
MarS-FL: A Market Share-based Decision Support Framework for Participation in Federated Learning
No Fear of Heterogeneity: Classifier Calibration for Federated Learning with Non-IID Data
On The Impact of Client Sampling on Federated Learning Convergence
OpenFed: A Comprehensive and Versatile Open-Source Federated Learning Framework
Optimal Model Averaging: Towards Personalized Collaborative Learning
Optimization-Based GenQSGD for Federated Edge Learning
Over-the-Air Aggregation for Federated Learning: Waveform Superposition and Prototype Validation
Personalized Federated Learning via Maximizing Correlation with Sparse and Hierarchical Extensions
Personalized Federated Learning with Gaussian Processes
Provably Secure Federated Learning against Malicious Clients
Qu-ANTI-zation: Exploiting Quantization Artifacts for Achieving Adversarial Outcomes
Revealing and Protecting Labels in Distributed Training
Robbing the Fed: Directly Obtaining Private Data in Federated Learning with Modified Models
Salvaging Federated Learning by Local Adaptation
Secure Sum Outperforms Homomorphic Encryption in (Current) Collaborative Deep Learning
Semi-Supervised Federated Learning with non-IID Data: Algorithm and System Design
SOAR: Minimizing Network Utilization with Bounded In-network Computing
Spatio-Temporal Federated Learning for Massive Wireless Edge Networks
Subgraph Federated Learning with Missing Neighbor Generation
The Skellam Mechanism for Differentially Private Federated Learning
To Talk or to Work: Delay Efficient Federated Learning over Mobile Edge Devices
Towards Model Agnostic Federated Learning Using Knowledge Distillation
What Do We Mean by Generalization in Federated Learning?
Wireless Federated Learning over MIMO Networks: Joint Device Scheduling and Beamforming Design
## 10.18 - 10.22
| Title |
| ----- |
A Federated Learning Aggregation Algorithm for Pervasive Computing: Evaluation and Comparison
Addressing Algorithmic Disparity and Performance Inconsistency in Federated Learning
BEV-SGD: Best Effort Voting SGD for Analog Aggregation Based Federated Learning against Byzantine Attackers
Bristle: Decentralized Federated Learning in Byzantine, Non-i.i.d. Environments
Decentralised Person Re-Identification with Selective Knowledge Aggregation
Deep-LIBRA: Artificial intelligence method for robust quantification of breast density with independent validation in breast cancer risk assessment
DistFL: Distribution-aware Federated Learning for Mobile Scenarios
Evaluation of Hyperparameter-Optimization Approaches in an Industrial Federated Learning System
Faster Non-Convex Federated Learning via Global and Local Momentum
Federated Graph Classification over Non-IID Graphs
Federated Learning over Wireless IoT Networks with Optimized Communication and Resources
Federated Multiple Label Hashing (FedMLH): Communication Efficient Federated Learning on Extreme Classification Tasks
Federated Reinforcement Learning: Techniques, Applications, and Open Challenges
Federated Test-Time Adaptive Face Presentation Attack Detection with Dual-Phase Privacy Preservation
Federated Unlearning via Class-Discriminative Pruning
FedGEMS: Federated Learning of Larger Server Models via Selective Knowledge Fusion
FedHe: Heterogeneous Models and Communication-Efficient Federated Learning
FedParking: A Federated Learning based Parking Space Estimation with Parked Vehicle assisted Edge Computing
Game of Gradients: Mitigating Irrelevant Clients in Federated Learning
Guess what? You can boost Federated Learning for free
Infinitely Divisible Noise in the Low Privacy Regime
Layer-wise Adaptive Model Aggregation for Scalable Federated Learning
MANDERA: Malicious Node Detection in Federated Learning via Ranking
Model-Based Reinforcement Learning Framework of Online Network Resource Allocation
On Seven Fundamental Optimization Challenges in Machine Learning
Optimal Client Sampling for Federated Learning
Over-the-Air Federated Multi-Task Learning
PipAttack: Poisoning Federated Recommender Systems forManipulating Item Promotion
PPSGCN: A Privacy-Preserving Subgraph Sampling Based Distributed GCN Training Method
PRECAD: Privacy-Preserving and Robust Federated Learning via Crypto-Aided Differential Privacy
Privacy-Preserving Federated Learning via Normalized (instead of Clipped) Updates
QLSD: Quantised Langevin stochastic dynamics for Bayesian federated learning
Robustness analytics to data heterogeneity in edge computing
Secure and Efficient Federated Learning Through Layering and Sharding Blockchain
SecureBoost+ : A High Performance Gradient Boosting Tree Framework for Large Scale Vertical Federated Learning
Semi-asynchronous Hierarchical Federated Learning for Cooperative Intelligent Transportation Systems
Tackling Dynamics in Federated Incremental Learning with Variational Embedding Rehearsal
TESSERACT: Gradient Flip Score to Secure Federated Learning Against Model Poisoning Attacks
Towards Federated Bayesian Network Structure Learning with Continuous Optimization
Towards General Deep Leakage in Federated Learning
TsmoBN: Interventional Generalization for Unseen Clients in Federated Learning
Universally consistent estimation of the reach
User-Centric Federated Learning
WebFed: Cross-platform Federated Learning Framework Based on Web Browser with Local Differential Privacy
## 10.11 - 10.15
| Title |
| ----- |
Learning to Coordinate in Multi-Agent Systems: A Coordinated Actor-Critic Algorithm and Finite-Time Guarantees
Adapt to Adaptation: Learning Personalization for Cross-Silo Federated Learning
An Information-Theoretic Analysis of The Cost of Decentralization for Learning and Inference Under Privacy Constraints
Application of Homomorphic Encryption in Medical Imaging
Blockchain and Federated Edge Learning for Privacy-Preserving Mobile Crowdsensing
Communication-Efficient Online Federated Learning Framework for Nonlinear Regression
Deep Federated Learning for Autonomous Driving
Distribution-Free Federated Learning with Conformal Predictions
Dual Attention-Based Federated Learning for Wireless Traffic Prediction
Evaluation of Hyperparameter-Optimization Approaches in an Industrial Federated Learning System
FedDQ: Communication-Efficient Federated Learning with Descending Quantization
Federated learning and next generation wireless communications: A survey on bidirectional relationship
Federated Learning for Big Data: A Survey on Opportunities, Applications, and Future Directions
Federated Learning for COVID-19 Detection with Generative Adversarial Networks in Edge Cloud Computing
Federated Learning Over Cellular-Connected UAV Networks with Non-IID Datasets
Federated Natural Language Generation for Personalized Dialogue System
Federated Route Leak Detection in Inter-domain Routing with Privacy Guarantee
FedMe: Federated Learning via Model Exchange
FedSEAL: Semi-Supervised Federated Learning with Self-Ensemble Learning and Negative Learning
FedSLD: Federated Learning with Shared Label Distribution for Medical Image Classification
FedSpeech: Federated Text-to-Speech with Continual Learning
Gradual Federated Learning with Simulated Annealing
Graph-Fraudster: Adversarial Attacks on Graph Neural Network Based Vertical Federated Learning
Homogeneous Learning : Self-Attention Decentralized Deep Learning 
Leveraging Spatial and Temporal Correlations in Sparsified Mean Estimation
Nothing Wasted: Full Contribution Enforcement in Federated Edge Learning
Partial Variable Training for Efficient On-Device Federated Learning 
Privacy-Preserving Phishing Email Detection Based on Federated Learning and LSTM
ProgFed: Effective, Communication, and Computation Efficient Federated Learning by Progressive Training
Resource Allocation for Simultaneous Wireless Information and Power Transfer Systems: A Tutorial Overview
Resource-constrained Federated Edge Learning with Heterogeneous Data: Formulation and Analysis
Sim-to-Real Transfer in Multi-agent Reinforcement Networking for Federated Edge Computing
The Skellam Mechanism for Differentially Private Federated Learning 
WAFFLE: Weighted Averaging for Personalized Federated Learning
## 10.04 - 10.08
| Title |
| ----- |
Asynchronous Upper Confidence Bound Algorithms for Federated Linear Bandits
Automatic Tuning of Federated Learning Hyper-Parameters from System Perspective
Blockchain-based Federated Learning : A Comprehensive Survey
Communication-Efficient Federated Learning with Binary Neural Networks
Distributed Learning Approaches for Automated Chest X-Ray Diagnosis
Efficient and Private Federated Learning with Partially Trainable Networks
Enabling On-Device Training of Speech Recognition Models with Federated Dropout
Exploring Heterogeneous Characteristics of Layers in ASR Models for More Efficient Training
FedDQ: Communication-Efficient Federated Learning with Descending Quantization
Federated Distillation of Natural Language Understanding with Confident Sinkhorns
Federated Distributionally Robust Optimization for Phase Configuration of RISs
Federated Learning for Big Data: A Survey on Opportunities, Applications, and Future Directions
Federated Learning from Small Datasets
Federated Learning via Plurality Vote
Federatingfor Learning Group Fair Models
FOD-A: A Dataset for Foreign Object Debris in Airports
FSL: Federated Supermask Learning 
MedPerf: Open Benchmarking Platform for Medical Artificial Intelligence using Federated Evaluation
Neural Tangent Kernel Empowered Federated Learning 
Privacy enabled Financial Text Classification using Differential Privacy and Federated Learning 
SecFL: Confidential Federated Learning using TEEs
Secure Aggregation for Buffered Asynchronous Federated Learning 
Secure Byzantine-Robust Distributed Learning via Clustering
Securing Federated Learning : A Covert Communication-based Approach
SSFL: Tackling Label Deficiency in Federated Learning via Personalized Self-Supervision
Towards Federated Learning -Enabled Visible Light Communication in 6G Systems
Two-Bit Aggregation for Communication Efficient and Differentially Private Federated Learning
